content/academy/apify_platform/getting_started/inputs_outputs.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Inputs & outputs
+description: Create an actor from scratch which takes an input, processes that input, then outputs a result that can be used elsewhere.
+menuWeight: 3
+paths:
+- apify-platform/getting-started/inputs-outputs
+---
+
+# [](#inputs-outputs) Inputs & outputs
+
+Most of the time, when you are writing any sort of software, it will generally expect some sort of input and generate some sort of output. It is the same exact story when it comes to actors, which is why we at Apify have made it so easy to accept input into an actor and store its results somewhere.
+
+In this lesson, we'll be demonstrating inputs and outputs by building an actor which takes two numbers as input, adds them up, then outputs the result.
+
+## [](#accept-input) Accept input into an actor
+
+Let's first create another new actor using the same template as before. Feel free to refer to the [previous lesson]({{@link apify_platform/getting_started/creating_actors.md}}) for a refresher on how to do this.
+
+Replace all of the code in **main.js** with this code snippet:
+
+```JavaScript
+const Apify = require('apify');
+
+Apify.main(async () => {
+    // Grab our numbers which were inputted
+    const { num1, num2 } = await Apify.getInput();
+    
+    // Calculate the solution
+    const solution = num1 + num2;
+
+    // Push the solution to the dataset
+    await Apify.pushData({ solution })
+});
+```
+
+Then, replace everything in **INPUT_SCHEMA.json**  with this:
```

Comment:
I would mention that this is optional (the actor will work without it) and that it generates a UI and validates input. As a new user, I would be surprised that I need to fill in such a big JSON just to accept input.